### PR TITLE
ICDS: Reverse order of aggregation

### PR DESCRIPTION
**Pull request by sheelio**
@emord 

This pulls out Kamil's change and reverses the order that months are run in.

### CODE REVIEW

File: custom/enikshay/const.py [hunk 1]
```diff
@@ -97,6 +97,7 @@
 )
 
 CASE_VERSION = 'case_version'
+CASE_MINOR_VERSION = 'case_minor_version'
```
Comment (emord) [inline custom/enikshay/const.py]: This is unrelated can you remove it?

**Comment by sheelio** [comment]
@emord 
Removed the file that got accidentally included.

**Review by emord — APPROVED** [review]
